articles/app-service/app-service-key-vault-references.md

```diff
@@ -32,7 +32,7 @@ In order to read secrets from Key Vault, you need to have a vault created and gi
 1. Create a [system-assigned managed identity](overview-managed-identity.md) for your application.
 
    > [!NOTE] 
-   > Key Vault references currently only support system-assigned managed identities. User-assigned identities cannot be used.
+   > Key Vault references currently only support system-assigned managed identities. User-assigned identities cannot be used for App Service / Azure Functions
```

Comment:
Please add back in the period that was removed.

The slash in this context feels odd. Perhaps we can replace that with "and" instead?

Reply:
Actually, I don't like how this reads. It implies that user-assigned can't be used in any context. What about moving the qualifying statement into the first sentence:

> Key Vault references for App Service and Azure Functions currently only support system-assigned managed identities. User-assigned identities cannot be used.